# 라이브러리

In [2]:
import pandas as pd

# RAW 데이터

In [40]:
df =pd.read_excel('2. 공공데이터 개방목록(2024.03월기준).xlsx')
df.columns

Index(['목록키', '기관구분', '기관명', '목록유형', '목록명', '목록설명', '조회수', '분류체계', '기관코드',
       '국가중점여부', '표준데이터여부', '목록 등록일', '목록 수정일', '목록 URL', '다운로드_활용건수', '키워드'],
      dtype='object')

In [41]:
drop_cols = ['목록유형', '기관코드', '국가중점여부', '표준데이터여부', '목록 등록일','목록 수정일', '목록 URL', '다운로드_활용건수']
df = df.drop(columns=drop_cols)
df.head()

목록키 기관구분  기관명                        목록명  \
0  15127225  교육부  교육부   교육부_특수교육 교육과정 총론 및 각론 고시   
1  15106239  교육부  교육부             교육부_특수교육 연차보고서   
2  15127207  교육부  교육부        교육부_재외교육기관포털_한국학교정보   
3  15102662  교육부  교육부       교육부_재외교육기관포털_한국교육원정보   
4  15126531  교육부  교육부  교육부_교육통계서비스_직업계고졸업자취업통계연보   

                                                목록설명   조회수            분류체계  \
0  교육부 고시 제2022-34호 / 이 교육과정은 유아교육법 제13조제2항, 초?중등...    60  교육 - 유아및초·중등교육   
1  특수교육의 개요, 특수교육 현황, 특수학교 현황, 일반학교 특수교육현황, 고등학교 ...   600       교육 - 교육일반   
2  재외 한국학교의 정의 : 재외국민에게 국내의 초·중등 교육과정에 따른 학교교육을 실...    34  교육 - 유아및초·중등교육   
3  재외교육기관포털에서 제공하고 있는 국가, 교육원명, 공관명, 도시, 원장, 주소, ...  1051       교육 - 교육일반   
4  직업계고 졸업자 취업통계연보는 직업계고 취업통계조사의 결과로서 직업계고 학생의 졸업...   161    교육 - 평생·직업교육   

                 키워드  
0       장애인,유치원,교육기회  
1  특수학교,장애인,특수교육주요시책  
2   학교주소,재외한국학교,동포자녀  
3    재외교육기관,설립일자,공관명  
4       학교수,졸업자,미취업자

In [21]:
for cat in df['분류체계'].unique():
    df_tmp = df[df['분류체계'] == cat].reset_index(drop=True)
    cat = cat.replace(' ', '')
    df_tmp.to_csv(f'{cat}.csv', index=False)

# 프롬프트

- 첨부 파일은 교육공공데이터 분석 활용대회에서 제공되는 데이터 목록이다. 

- 교육공공데이터 분석 활용대회란, 교육 공공데이터 이용을 활성화하고 학생들의 데이터 문제 해결능력(리터러시) 향상과 창업 아이디어, 교육정책을 발굴하기 위해 매년 개최되고 있다. 
- 교육 공공 데이터란, 교육보, 시도 교육청, 교육유관기관에 축적되어 있는 공적인 교육관련 데이터 이다. 데이터 분석시에는, 이와 같은 교육 공공데이터를 활용해 데이터 목적에 부합한 해석 및 문제 제안, 그리고 문제의 해결 방안을 제시하며, 분석외에 새로운 지표 또는 새로운 관점의 분석 내용의 제안을 하는 것이다. 

- 너는 중학교 1학년인 대회의 참가자가 되어, 제공되는 파일을 보고, 
    - 데이터 분석을 하기에 가장 적절한 아이디어 3개와, 
    - 이와 관련된 데이터의 목록키와 목록명을 알려줘. 

- 아이디어는 되도록 자세하게 (아이디어의 설명, 아이디어의 우수한 점, 아이디어 구현의 문제점)를 제공해줘. 주의, 중학교 1학년 수준에 맞는 쉬운 아이디어이면서 참신하고 재미 있는 아이디어를 제공해줘.

# GPT 결과

In [2]:
import os
file_list = os.listdir()
file_list.sort()

In [3]:
cat_list = []
for file in file_list:
    if 'csv' in file:
        cat_list.append(file.split('.csv')[0])

In [5]:
for val in cat_list:
    print(val, end=', ')

공공질서및안전-안전관리, 과학기술-과학기술연구, 과학기술-과학기술진흥, 과학기술-원자력기술, 교육-고등교육, 교육-교육일반, 교육-유아및초·중등교육, 교육-평생·직업교육, 교통및물류-도로, 문화체육관광-문화예술, 문화체육관광-문화재, 문화체육관광-문화체육관광일반, 문화체육관광-체육, 사회복지-고용노동, 사회복지-공적연금, 사회복지-보육·가족및여성, 사회복지-사회복지일반, 사회복지-주택, 사회복지-취약계층지원, 일반공공행정-법제, 일반공공행정-일반행정, 일반공공행정-지방행정·재정지원, 통신-방송통신, 통일·외교-외교, 해양수산-해양수산·어촌, 환경-대기, 환경-폐기물, 

## 공공질서및안전-안전관리

In [3]:
filename = '공공질서및안전-안전관리'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 학교 주변 CCTV 탐험 게임
- **아이디어 설명**: 학교 주변의 CCTV 위치를 지도에 표시하고, 학생들이 이를 찾아보는 탐험 게임을 만듭니다. 이 게임은 학교 안전을 학습하면서 동시에 지리적 인식 능력을 키울 수 있는 기회를 제공합니다.
- **아이디어의 우수한 점**: 학생들이 자신의 학교 주변 환경에 대해 더 잘 알게 되고, 안전 인식을 자연스럽게 키울 수 있습니다.
- **아이디어 구현의 문제점**: 모든 학생이 게임에 참여하려면 스마트폰 또는 태블릿과 같은 디지털 기기가 필요할 수 있으며, 위치 데이터의 정확성이 게임의 성공에 중요합니다.
- **데이터**: 목록키 15095755, 목록명 "경상북도교육청 경상북도교육청정보센터_CCTV설치 정보 개방"

### 2. 내 학교 CCTV 설치 비교 분석
- **아이디어 설명**: 경상북도교육청의 다른 학교들과 비교하여 우리 학교의 CCTV 설치 상황을 분석합니다. 이를 통해 학생들이 데이터 비교와 분석 기법을 배울 수 있습니다.
- **아이디어의 우수한 점**: 데이터를 이용하여 실제로 자신의 학교가 어떻게 다른지 이해하는 데 도움을 주며, 분석적 사고를 촉진합니다.
- **아이디어 구현의 문제점**: 데이터 분석 도구나 기법을 처음 접하는 학생들에게는 어려울 수 있으며, 선생님의 지도가 필수적일 수 있습니다.
- **데이터**: 목록키 15095755, 목록명 "경상북도교육청 경상북도교육청정보센터_CCTV설치 정보 개방"

### 3. 안전한 학교 만들기 프로젝트
- **아이디어 설명**: 학교 내외의 CCTV 설치 정보를 바탕으로 학생들이 자신의 학교를 더 안전하게 만들기 위한 제안을 합니다. 학생들은 설문조사를 통해 어떤 지역에 더 많은 CCTV가 필요한지 의견을 모으고, 이를 바탕으로 개선안을 제안합니다.
- **아이디어의 우수한 점**: 학생들이 직접 자신의 학교 안전에 참여함으로써 책임감을 느끼고, 문제 해결 능력을 키울 수 있습니다.
- **아이디어 구현의 문제점**: 모든 학생의 의견을 수렴하고 조정하는 과정에서 시간과 노력이 많이 들 수 있습니다.
- **데이터**: 목록키 15095755, 목록명 "경상북도교육

## 과학기술-과학기술연구

In [4]:
filename = '과학기술-과학기술연구'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 과학기술 교육 자료의 효과성 분석
- **아이디어 설명**: 한국과학창의재단의 "SWAI교재" 데이터를 사용하여, 이 교재가 학생들의 학습 효과에 얼마나 기여하는지 분석합니다. 데이터를 통해 학생들의 성적 향상, 관심도 증가 등의 지표를 측정할 수 있습니다.
- **아이디어의 우수한 점**: 교육 자료의 효과를 실제 데이터로 분석함으로써 교육 정책 개선에 도움을 줄 수 있습니다.
- **아이디어 구현의 문제점**: 학생들의 개인 정보 보호와 데이터 접근성에 제한이 있을 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15121104
- 목록명: 한국과학창의재단_SWAI교재

### 2. 국민의 과학기술 이해도 변화 추적
- **아이디어 설명**: "과학기술국민이해도 조사보고서" 데이터를 활용하여 시간에 따른 국민의 과학기술 이해도 변화를 분석합니다. 이를 통해 과학 교육의 변화 필요성을 제시할 수 있습니다.
- **아이디어의 우수한 점**: 국민 교육 정책에 직접적인 영향을 줄 수 있는 중요한 데이터 기반 의사결정을 제공합니다.
- **아이디어 구현의 문제점**: 보고서의 데이터가 다소 오래되었거나 불완전할 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 3071313
- 목록명: 한국과학창의재단_과학기술국민이해도 조사보고서

### 3. 디지털 교육 공간의 이용 현황과 효과 분석
- **아이디어 설명**: "스마트 수과학실" 데이터를 활용하여 디지털 교육 공간의 이용 현황과 그 효과를 분석합니다. 이를 통해 향후 디지털 교육 자원의 배분과 개선 방향을 제시할 수 있습니다.
- **아이디어의 우수한 점**: 신기술을 교육에 효과적으로 적용하는 방법을 찾아낼 수 있습니다.
- **아이디어 구현의 문제점**: 사용자 데이터의 접근성과 정확성에 의존해야 하며, 실제 학습 효과를 측정하기 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15121150
- 목록명: 한국과학창

## 과학기술-과학기술진흥

In [5]:
filename = '과학기술-과학기술진흥'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. SW 중심 사회의 사업 성과 분석
- **아이디어 설명**: "SW중심사회 사업성과물" 데이터를 활용하여 소프트웨어 교육 및 관련 사업들이 어떻게 사회에 영향을 미치고 있는지 분석합니다. 프로젝트의 성과, 참여도, 그리고 이들이 학생 및 일반인에게 끼친 영향을 살펴봅니다.
- **아이디어의 우수한 점**: 소프트웨어 중심 사회로의 전환을 위한 정책과 프로그램의 효과를 검증할 수 있습니다.
- **아이디어 구현의 문제점**: 데이터가 포괄적이지 않을 수 있으며, 사업 성과의 정확한 영향을 측정하기 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15121091
- 목록명: 한국과학창의재단_SW중심사회 사업성과물

### 2. 온라인 코딩 교육의 효과 분석
- **아이디어 설명**: "온라인 코딩파티 참가자 수 및 이수자 수" 데이터를 분석하여 온라인 코딩 교육 프로그램의 효과와 참여도를 평가합니다. 이 프로그램이 얼마나 많은 학생들을 끌어들였는지, 완료율은 어떤지 분석합니다.
- **아이디어의 우수한 점**: 디지털 교육이 학생들에게 미치는 실질적인 영향을 이해하고, 교육 방식의 개선을 도모할 수 있습니다.
- **아이디어 구현의 문제점**: 참가자의 배경 정보나 학습 전후의 성과를 자세히 알기 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15091315
- 목록명: 한국과학창의재단_온라인 코딩파티 참가자 수 및 이수자 수

### 3. 창의적 체험활동 프로그램의 효과성 분석
- **아이디어 설명**: "창의적 체험활동 프로그램 정보" 데이터를 활용하여 이 프로그램이 학생들의 창의성과 학습 동기에 어떤 영향을 미쳤는지 분석합니다. 프로그램 참여 학생들의 학습 성과나 창의력 향상을 측정할 수 있습니다.
- **아이디어의 우수한 점**: 교육 프로그램의 질적 향상과 정책 개발에 기여할 수 있는 근거를 제공합니다.
- **아이디어 구현의 문제점**: 창의적 체험활동의 효과를 정확히 측정하기 위해서는 다양한 변수(예: 학생의 기본 학업 성취도, 가정 환경 등)를 고려해야 하며, 이를 위한 데이터 수집이 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15091154
- 목록명: 한국과학창의재단_창의적 체험활동 프로그램 정보


## 과학기술-원자력기술

In [6]:
filename = '과학기술-원자력기술'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 원자력진흥위원회의 역할과 영향 분석
- **아이디어 설명**: "원자력진흥 위원회 위원 관리" 데이터를 사용하여 원자력진흥위원회의 위원 구성과 그들의 전문성이 원자력 정책과 프로젝트에 어떻게 영향을 미치는지 분석합니다. 위원들의 배경과 경력을 연구하여, 위원회의 다양성과 전문성이 정책 결정에 어떻게 기여하는지 살펴봅니다.
- **아이디어의 우수한 점**: 이 분석은 공공 정책의 투명성을 높이고, 전문가 그룹의 구성이 어떻게 중요한 정책 결정에 영향을 미치는지 이해하는 데 도움이 됩니다.
- **아이디어 구현의 문제점**: 위원의 세부적인 정보 접근이 제한될 수 있으며, 개인 정보 보호 규정을 준수해야 합니다.

**데이터 목록키와 목록명**
- 목록키: 15117391
- 목록명: 한국연구재단_원자력진흥 위원회 위원 관리


### 2. 원자력 기술 교육 프로그램의 발전 분석
- **아이디어 설명**: 원자력 기술 관련 교육 프로그램이 어떻게 발전해 왔는지 탐색합니다. 이 데이터를 통해 원자력 교육이 중점을 두고 있는 영역과 시간에 따른 교육 내용의 변화를 분석할 수 있습니다.
- **아이디어의 우수한 점**: 원자력 교육의 변화를 이해함으로써 향후 교육 정책 개발에 기여할 수 있습니다.
- **아이디어 구현의 문제점**: 현재 데이터에서는 원자력 교육 프로그램에 대한 직접적인 데이터가 부족할 수 있으므로, 관련 데이터를 추가로 수집할 필요가 있습니다.

### 3. 원자력 안전 규제와 기술 발전의 상관관계 분석
- **아이디어 설명**: 원자력 기술의 발전이 원자력 안전 규제에 어떤 영향을 미쳤는지 분석합니다. 기술 발전이 규제 변경에 미친 영향을 시간 순으로 정리하고, 주요 사건이나 사고에 따른 규제 변화를 조사합니다.
- **아이디어의 우수한 점**: 이런 분석은 과학기술의 발전이 공공 정책에 어떤 실질적인 영향을 미치는지를 이해하는 데 중요합니다.
- **아이디어 구현의 문제점**: 공개된 데이터의 범위가 제한적일 수 있으며, 정책적 변화와 기술 발전 간의 직접적인 연결고리를 찾기 어려울 수 있습니다.

## 교육-고등교육

In [7]:
filename = '교육-고등교육'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 대학원 계약학과의 설치 및 운영 분석
- **아이디어 설명**: "대학원 계약학과 설치 운영 현황_학교별" 데이터를 활용하여, 국내 대학원들이 어떤 계약학과를 운영하고 있는지, 이들 학과의 지역별, 대학별 분포를 분석합니다. 계약학과의 특성과 이들 학과가 산업에 끼치는 영향도 조사합니다.
- **아이디어의 우수한 점**: 교육 정책 결정자와 학생들에게 유용한 정보를 제공하여, 대학원 교육의 방향성을 결정하는 데 도움을 줄 수 있습니다.
- **아이디어 구현의 문제점**: 각 학과의 상세 데이터 접근성과 정보의 최신성을 확보해야 합니다.

**데이터 목록키와 목록명**
- 목록키: 15122460
- 목록명: 교육부_대학원 계약학과 설치 운영 현황_학교별

### 2. 대학 주요 정보 및 학과 성과 분석
- **아이디어 설명**: "대학알리미_대학주요정보_학생_교원_연구_재정_교육여건" 데이터를 분석하여 각 대학의 학생 수, 교원 수, 연구 성과, 재정 상태 및 교육 여건을 비교합니다. 이를 통해 대학별 교육의 질과 효율성을 평가할 수 있습니다.
- **아이디어의 우수한 점**: 대학 선택에 있어서 중요한 정보를 제공하고, 교육의 질을 객관적으로 평가할 수 있습니다.
- **아이디어 구현의 문제점**: 데이터의 포괄성과 정확성을 확인해야 하며, 다양한 변수 간의 상관관계를 정확히 분석하기 위한 통계적 방법론이 필요합니다.

**데이터 목록키와 목록명**
- 목록키: 15118998
- 목록명: 교육부_대학알리미_대학주요정보_학생_교원_연구_재정_교육여건

### 3. 특성화학과 및 학과 선택 트렌드 분석
- **아이디어 설명**: "대학알리미_특성화학과 분야 정보" 데이터를 사용하여 최근 몇 년간 대학들이 어떤 특성화학과를 운영하고 있는지 분석합니다. 학과 선택의 트렌드와 시대별 변화를 살펴보며, 인기 있는 학과와 그 원인을 탐색합니다.
- **아이디어의우수한 점**: 학생들의 학과 선택 패턴을 이해하고, 교육 정책과 대학 프로그램 개발에 있어 중요한 인사이트를 제공합니다.
- **아이디어 구현의 문제점**: 특성화 학과의 성과 측정과 인기도 평가를 위한 추가적인 데이터가 필요할 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15119000
- 목록명: 교육부_대학알리미_특성화학과 분야 정보

## 교육-교육일반

In [8]:
filename = '교육-교육일반'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 특수교육 연차보고서를 통한 특수교육 발전 추적
- **아이디어 설명**: "특수교육 연차보고서" 데이터를 활용하여 연도별로 특수교육의 발전 상황을 추적합니다. 이 데이터를 통해 특수학교의 수, 일반학교 내 특수교육 현황 등을 분석하고, 시간이 지남에 따른 특수교육 정책의 변화와 효과를 평가합니다.
- **아이디어의 우수한 점**: 특수교육에 대한 정부의 노력과 성과를 명확하게 보여주며, 정책 개선에 대한 구체적인 근거를 제공할 수 있습니다.
- **아이디어 구현의 문제점**: 데이터의 일관성과 완성도를 확보하기 위해 추가적인 데이터 정제와 분석이 필요할 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15106239
- 목록명: 교육부_특수교육 연차보고서

### 2. 재외교육기관의 현황과 영향 분석
- **아이디어 설명**: "재외교육기관포털_한국교육원정보" 데이터를 활용하여 전 세계의 한국교육원의 분포, 규모 및 활동을 분석합니다. 각 국가별 한국교육원의 역할과 교육 프로그램의 효과를 조사하여, 글로벌 한국어 교육의 영향력을 평가합니다.
- **아이디어의 우수한 점**: 한국어 및 한국 문화 교육의 국제적 확산을 이해하고, 해외에서의 한국 교육 정책의 효과를 평가할 수 있습니다.
- **아이디어 구현의 문제점**: 해외 기관들의 다양한 행정적, 문화적 환경을 고려해야 하며, 정보 수집에 한계가 있을 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15102662
- 목록명: 교육부_재외교육기관포털_한국교육원정보

### 3. 교육부 사업 공고 분석을 통한 정책 변화 추적
- **아이디어 설명**: "대표홈페이지 사업공고 목록" 데이터를 활용하여 교육부에서 발표하는 다양한 교육 관련 사업들의 추세와 변화를 분석합니다. 연도별로 발표된 사업들의 주제와 범위를 조사하여 교육 정책의 방향과 중점을 평가합니다. 이 분석을 통해 특정 시기에 어떤 교육 이슈가 중요하게 다뤄졌는지 이해할 수 있습니다.
- **아이디어의 우수한 점**: 교육 정책의 변화와 시대적 요구를 반영하는 사업의 동향을 파악함으로써, 향후 교육 계획과 개선 방향에 대한 근거를 마련할 수 있습니다.
- **아이디어 구현의 문제점**: 사업 공고만으로는 실제 실행된 사업의 성과나 영향을 평가하기 어려울 수 있으므로, 추가적인 성과 데이터가 필요할 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15124462
- 목록명: 교육부_대표홈페이지 사업공고 목록

## 교육-유아및초·중등교육

In [9]:
filename = '교육-유아및초·중등교육'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 특수교육 교육과정의 변화와 영향 분석
- **아이디어 설명**: "특수교육 교육과정 총론 및 각론 고시" 데이터를 활용하여, 특수교육 교육과정의 변화를 연도별로 추적하고, 이러한 변화가 학생들에게 어떤 영향을 미쳤는지 분석합니다. 교육과정의 개정 사항과 그 결과로 나타난 교육적 성과나 학생들의 반응을 조사합니다.
- **아이디어의 우수한 점**: 특수교육이 시대의 요구와 학생들의 필요에 어떻게 부응하고 있는지 이해할 수 있습니다.
- **아이디어 구현의 문제점**: 교육 과정의 변화와 학생들의 성과 사이의 직접적인 인과 관계를 입증하기 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15127225
- 목록명: 교육부_특수교육 교육과정 총론 및 각론 고시

### 2. 재외 한국학교의 교육 현황 분석
- **아이디어 설명**: "재외교육기관포털_한국학교정보" 데이터를 활용하여 전 세계에 분포한 한국학교의 교육 현황을 분석합니다. 학교의 수, 위치, 제공하는 교육 프로그램의 종류 등을 조사하고, 이들 학교가 재외 교육 공동체에 끼치는 영향을 평가합니다.
- **아이디어의 우수한 점**: 글로벌 한국 교육 네트워크의 현재 상태와 발전 가능성을 파악할 수 있습니다.
- **아이디어 구현의 문제점**: 각국의 교육 법규와 문화적 차이를 고려한 복잡한 분석이 필요할 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15127207
- 목록명: 교육부_재외교육기관포털_한국학교정보

### 3. 전국 진로상담교사 배치 현황과 효과 분석
- **아이디어 설명**: "교육부 시도별 진로상담교사 배치현황" 데이터를 활용하여 각 시도별로 진로상담교사가 어떻게 배치되어 있는지 분석합니다. 진로상담교사의 배치와 학생들의 진로 결정, 학업 성취도와의 상관관계를 조사하여, 진로 지도의 효과를 평가합니다.
- **아이디어의우수한 점**: 진로상담의 질과 범위가 학생들의 교육적 결과와 직업적 방향에 미치는 영향을 이해할 수 있어, 교육 정책 개선에 중요한 데이터를 제공합니다.
- **아이디어 구현의 문제점**: 진로상담의 효과를 측정하기 위해서는 학생들의 장기적인 교육 및 직업 추적 데이터가 필요하며, 이를 수집하는 것은 도전적일 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15126419
- 목록명: 교육부_교육부 시도별 진로상담교사 배치현황

## 교육-평생·직업교육

In [10]:
filename = '교육-평생·직업교육'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 직업계고 졸업자의 취업 성공률 분석
- **아이디어 설명**: "교육부_교육통계서비스_직업계고졸업자취업통계연보" 데이터를 활용하여, 직업계고 졸업자의 취업률을 분석합니다. 취업률의 연도별 변화, 학교별 취업 성공률, 그리고 졸업 후 진로의 분야별 분포를 조사합니다.
- **아이디어의 우수한 점**: 직업교육의 효과성을 평가하고, 교육 정책 개선에 필요한 실질적인 데이터를 제공합니다.
- **아이디어 구현의 문제점**: 취업률 데이터만으로는 취업의 질과 지속 가능성을 평가하기 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15126531
- 목록명: 교육부_교육통계서비스_직업계고졸업자취업통계연보

### 2. 진로체험 프로그램의 효과성 분석
- **아이디어 설명**: "진로체험망_진로체험프로그램현황" 데이터를 활용하여 전국의 진로체험 프로그램들의 범위와 종류를 분석하고, 이 프로그램들이 학생들의 진로 선택에 미치는 영향을 평가합니다.
- **아이디어의 우수한 점**: 학생들에게 제공되는 진로체험 기회의 질과 범위를 이해하고, 교육 기회의 균등성을 높이는 방향을 제시할 수 있습니다.
- **아이디어 구현의 문제점**: 프로그램의 직접적인 효과를 측정하기 위해 학생들의 피드백과 장기적인 진로 데이터가 필요합니다.

**데이터 목록키와 목록명**
- 목록키: 15123664
- 목록명: 교육부_진로체험망_진로체험프로그램현황

### 3. 학위수여 교육훈련기관의 교육 기회 분석
- **아이디어 설명**: "대학의 장에 의한 학위수여 교육훈련기관 목록" 데이터를 활용하여 평생교육원 및 대학 부설 교육기관에서 제공하는 학위 및 교육 프로그램을 분석합니다. 각 기관이 제공하는 학위 종류와 프로그램의 특성을 조사하여, 평생교육의 접근성과 다양성을 평가합니다.
- **아이디어의 우수한 점**: 평생교육 시스템의 구성과 효과성을 평가하여, 교육 기회 확대 및 직업 훈련 프로그램 개선에 도움을 줄 수 있습니다.
- **아이디어 구현의 문제점**: 개별 교육 기관의 프로그램 세부사항 및 성과 데이터가 불완전하거나 접근이 제한적일 수 있어, 전체적인 효과 분석에 한계가 있을 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15122224
- 목록명: 교육부_대학의 장에 의한 학위수여 교육훈련기관 목록

##  교통및물류-도로

In [11]:
filename = '교통및물류-도로'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 자율주행 차량의 데이터 수집 및 분석
- **아이디어 설명**: "서울대학교 연구소 차세대융합기술연구원_판교제로시티 자율주행 데이터 조회 서비스" 데이터를 활용하여 자율주행 차량의 운행 데이터를 분석합니다. 데이터에는 차량의 위치, 속도, 주행 경로, 주변 환경 데이터 등이 포함되어 있을 수 있습니다. 이 데이터를 분석하여 자율주행 기술의 안정성, 효율성을 평가하고, 교통 흐름에 미치는 영향을 조사합니다.
- **아이디어의 우수한 점**: 자율주행 기술의 발전에 필요한 실제 운행 데이터를 통해 기술적 문제점을 파악하고 개선 방안을 제시할 수 있습니다.
- **아이디어 구현의 문제점**: 데이터의 복잡성과 대용량 처리를 필요로 하며, 데이터의 보안과 개인정보 보호 문제를 고려해야 합니다

### 2. 판교 제로시티 프로젝트의 교통 효율성 분석
- **아이디어 설명**: 판교 제로시티 프로젝트의 교통 데이터를 사용하여 프로젝트 구역 내의 교통 흐름과 자율주행 셔틀의 통합이 교통 혼잡 및 안전에 미치는 영향을 분석합니다. 이를 통해 자율주행 차량이 기존 교통 시스템에 통합될 때의 이점과 문제점을 평가합니다.
- **아이디어의 우수한 점**: 자율주행 차량의 도입이 실제 도시 환경에서 어떻게 교통 문제를 완화하거나 악화시킬 수 있는지에 대한 실증적인 분석을 제공합니다.
- **아이디어 구현의 문제점**: 효과적인 분석을 위해서는 교통사고, 대기시간, 경제적 비용 등 다양한 변수의 데이터가 필요하며, 이를 통합 분석하기 위한 복잡한 모델링이 요구됩니다.

### 3. 자율주행 기술의 사회적 수용성 평가
- **아이디어 설명**: 자율주행 차량과 관련된 데이터를 활용하여 대중의 자율주행 기술에 대한 인식과 수용성을 평가합니다. 데이터에서 차량의 운행 패턴, 사고 발생 빈도, 대중의 피드백 등을 분석하여 기술의 사회적 수용도를 조사합니다.
- **아이디어의 우수한 점**: 자율주행 기술에 대한 대중의 신뢰와 수용도를 이해함으로써, 기술 개발자와 정책 입안자에게 필요한 정보를 제공할 수 있습니다. 이는 기술적 개선뿐만 아니라, 사회적, 규제적 차원에서의 조치를 계획하는 데 도움을 줄 수 있습니다.
- **아이디어 구현의 문제점**: 사회적 수용성을 측정하기 위해서는 설문조사나 인터뷰 등 추가적인 사회과학적 연구 방법론을 통합해야 하며, 이러한 데이터의 수집과 분석은 시간과 자원이 많이 소모될 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15098901
- 목록명: 서울대학교 연구소 차세대융합기술연구원_판교제로시티 자율주행 데이터 조회 서비스

## 문화체육관광-문화예술

In [12]:
filename = '문화체육관광-문화예술'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 지역 도서관 도서 이용 패턴 분석
- **아이디어 설명**: "월별최다대출" 데이터를 활용하여, 경상북도교육청 구미도서관의 월별 최다 대출 도서를 분석합니다. 이를 통해 독자들의 독서 선호도와 계절별, 월별 독서 패턴을 파악할 수 있습니다.
- **아이디어의 우수한 점**: 지역 커뮤니티의 문화 소비 패턴을 이해하고, 도서관 서비스와 도서 구입 계획을 최적화할 수 있습니다.
- **아이디어 구현의 문제점**: 대출 데이터만으로는 독서 선호의 모든 측면을 포괄적으로 이해하기 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 3068862
- 목록명: 경상북도교육청 경상북도교육청구미도서관_월별최다대출

### 2. 희망도서 요청 분석을 통한 도서 구입 가이드라인 제안
- **아이디어 설명**: "희망도서구입목록" 데이터를 분석하여 도서관 이용자들이 요청한 희망도서 정보를 분석합니다. 이 데이터를 사용하여 도서 선택의 트렌드를 파악하고, 향후 도서 구매 계획을 수립하는 데 도움을 줄 수 있습니다.
- **아이디어의 우수한 점**: 이용자들의 요구와 선호에 기반한 데이터 기반 도서 구매 결정으로, 도서관 자원을 효율적으로 관리할 수 있습니다.
- **아이디어 구현의 문제점**: 희망도서 요청은 사용자의 요구를 반영하지만, 구매 예산과 우선 순위에 따라 선택이 필요할 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 3068859
- 목록명: 경상북도교육청 경상북도교육청구미도서관_희망도서구입목록

### 3. 도서관 도서 목록 및 추천 시스템 평가
- **아이디어 설명**: "도서 목록" 및 "추천도서" 데이터를 활용하여, 경상북도교육청 구미도서관에서 제공하는 도서 목록의 다양성과 추천도서의 적합성을 분석합니다. 이를 통해 도서관의 도서 구성과 추천 시스템의 효과성을 평가할 수 있습니다.
- **아이디어의 우수한 점**: 도서관 서비스의 질을 개선하고, 이용자 만족도를 높이는 데 기여할 수 있습니다.
- **아이디어 구현의 문제점**: 도서의 추천 및 선택에는 개인의 취향이 크게 작용하기 때문에, 모든 이용자의 만족을 보장하기 어려울 수 있습니다. 또한, 추천 시스템의 평가를 위해서는 사용자 피드백 수집이 중요하며, 이를 위한 추가적인 조사가 필요할 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 3068857
- 목록명: 경상북도교육청 경상북도교육청구미도서관_도서정보

## 문화체육관광-문화재

In [13]:
filename = '문화체육관광-문화재'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 고구려 벽화의 문화적 요소 분석
- **아이디어 설명**: "강서대묘_동남벽의 제3굄돌(옆면의 좌측 봉황 상세도)" 데이터를 활용하여 고구려 벽화에 표현된 문화적 요소를 분석합니다. 벽화에 나타난 상징물, 동식물, 인물의 표현 방식을 연구하여 고구려 사회의 신념과 예술적 가치를 조명합니다.
- **아이디어의 우수한 점**: 고대 역사 속 예술작품을 통해 당시 사회의 문화적 배경과 세계관을 이해할 수 있습니다.
- **아이디어 구현의 문제점**: 고대 벽화의 해석은 주관적 요소가 강하고, 다양한 해석이 가능하기 때문에 명확한 결론을 도출하기 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 2647202
- 목록명: 동북아역사재단_강서대묘_동남벽의 제3굄돌(옆면의 좌측 봉황 상세도)

### 2. 중앙아시아 암각화의 비교 분석
- **아이디어 설명**: "후렝 우주르 하단 올_소와 말 그리고 산양(후렝 우주르 하단 올 7구)" 데이터를 활용하여 중앙아시아의 암각화와 한반도의 선사 암각화를 비교 분석합니다. 두 지역의 암각화에서 나타나는 공통적인 요소와 지역적 특성을 분석하여, 문화적 교류와 독립적 발전의 특징을 파악합니다.
- **아이디어의 우수한 점**: 문화재 연구에 있어서 비교 분석은 광범위한 문화적 영향력과 교류를 이해하는 데 도움을 줍니다.
- **아이디어 구현의 문제점**: 서로 다른 지역의 예술 작품을 비교하는 과정에서 문화적 맥락과 시대적 배경의 올바른 해석이 필요합니다.

**데이터 목록키와 목록명**
- 목록키: 2591771
- 목록명: 동북아역사재단_후렝 우주르 하단 올_소와 말 그리고 산양(후렝 우주르 하단 올 7구)

### 3. 암각화의 문화적 가치와 보존 문제
- **아이디어 설명**: "하난하드-산양들 4" 데이터를 분석하여 암각화의 문화적 가치와 현재 보존 상태를 조사합니다. 이를 통해 암각화가 직면한 자연적, 인위적 위협을 식별하고, 효과적인 보존 전략을 제안합니다.
- **아이디어의 우수한 점**: 암각화와 같은 문화재의 보존은 역사적 유산을 후대에 전달하는 중요한 작업입니다. 이 분석을 통해 보존 필요성과 방법을 명확히 할 수 있습니다.
- **아이디어 구현의 문제점**: 현장 접근성의 제한, 보존 상태 평가의 전문성 요구, 그리고 환경적 변화에 대한 지속적인 모니터링이 필요합니다.

**데이터 목록키와 목록명**
- 목록키: 2155255
- 목록명: 동북아역사재단_하난하드-산양들 4

## 문화체육관광-문화체육관광일반

In [14]:
filename = '문화체육관광-문화체육관광일반'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 고문헌자료를 통한 역사적 사건 및 문화 분석
- **아이디어 설명**: "고문헌자료 문헌정보서비스" 데이터를 활용하여, 고문헌에 기록된 역사적 사건, 인물, 문화적 요소를 분석합니다. 이 자료를 통해 시대별, 지역별 역사적 배경과 사회상을 연구하고 문화적 변화를 추적합니다.
- **아이디어의 우수한 점**: 고문헌은 시대의 생생한 기록을 담고 있어, 역사와 문화 연구에 깊이를 더하고, 보다 정확한 역사적 이해를 가능하게 합니다.
- **아이디어 구현의 문제점**: 고문헌의 해석에는 전문적 지식이 필요하며, 문자 자료의 오역이나 해석 차이로 인해 정보의 정확성에 영향을 미칠 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15057812
- 목록명: 한국학중앙연구원_고문헌자료 문헌정보서비스

### 2. 고지도를 활용한 지리적 변화 연구
- **아이디어 설명**: "고지도 고지명서비스" 데이터를 활용하여 고지도에 나타난 지리적 특성과 지명의 변화를 연구합니다. 이를 통해 특정 지역의 역사적 변천사 및 지리적 변화를 추적하고, 과거 지리적 환경이 현재에 끼친 영향을 분석합니다.
- **아이디어의 우수한 점**: 과거와 현재의 지리적 상태를 비교함으로써, 자연 및 인문 환경 변화의 원인과 결과를 이해할 수 있습니다.
- **아이디어 구현의 문제점**: 고지도의 정확성과 현대 지리 데이터와의 일치성을 확인하는 데 도전이 있을 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15059485
- 목록명: 한국학중앙연구원_고지도 고지명서비스

### 3. 고문헌 및 고지도의 디지털 보존과 활용
- **아이디어 설명**: 한국학중앙연구원에서 제공하는 고문헌 및 고지도 자료의 디지털 보존 상태와 활용 방안을 분석합니다. 이 자료의 디지털화 과정, 접근성, 사용자 인터페이스를 평가하고, 이를 개선하기 위한 전략을 제안합니다.
- **아이디어의 우수한 점**: 디지털화는 문화유산의 보존과 보다 넓은 접근성을 제공하며, 교육 및 연구 자원으로의 활용을 증대시킵니다.
- **아이디어 구현의 문제점**: 기술적 요구 사항과 비용이 많이 들 수 있으며, 디지털 보존 과정에서 발생할 수 있는 데이터 손실 또는 왜곡에 대한 주의가 필요합니다.

**데이터 목록키와 목록명**
- 데이터 목록키: 해당 데이터에 대한 목록키 정보는 제공되지 않았습니다.
- 목록명: 이 아이디어는 기존의 목록명을 활용하지 않고, 한국학중앙연구원에서 제공하는 고문헌 및 고지도 자료의 디지털 보존 전반에 관한 분석을 포함합니다.

이 세 가지 아이디어는 한국학중앙연구원의 자료를 활용하여 고문헌과 고지도의 학문적 및 사회적 가치를 연구하고, 이를 현대적 방식으로 보존하고 활용하는 방안을 탐색합니다. 각 아이디어는 문화유산의 디지털 보존, 교육적 활용, 그리고 공공 접근성 향상을 목표로 합니다. 이를 통해 역사적 유산의 가치를 현대 사회에 재조명하고, 더욱 효과적으로 보존하고 활용할 수 있는 방안을 모색할 수 있습니다.

## 문화체육관광-체육

In [15]:
filename = '문화체육관광-체육'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 공동체육관 프로그램 이용 분석
- **아이디어 설명**: "부산광역시교육청 부산광역시해운대교육지원청_공동체육관 정보" 데이터를 활용하여, 공동체육관에서 제공하는 다양한 스포츠 프로그램(배드민턴, 스쿼시, 헬스 등)의 참여율과 인기도를 분석합니다. 이를 통해 지역사회 내에서 스포츠 프로그램의 수요와 선호도를 파악합니다.
- **아이디어의 우수한 점**: 지역사회의 체육 활동 참여 동향을 이해하고, 프로그램 제공의 효율성을 높이는 데 기여할 수 있습니다.
- **아이디어 구현의 문제점**: 프로그램별 참여 데이터의 정확성과 완성도가 데이터 분석의 품질에 크게 영향을 미칠 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15023796
- 목록명: 부산광역시교육청 부산광역시해운대교육지원청_공동체육관 정보

### 2. 체육 프로그램의 계절별 변화와 코로나19 영향 분석
- **아이디어 설명**: 동일 데이터를 사용하여 체육 프로그램의 계절별 참여 변화를 분석하고, 코로나19의 발발 이후 이용 패턴의 변화를 조사합니다. 특히 팬데믹 기간 동안의 스포츠 활동 제한과 그 이후 회복 과정을 분석하여, 보건 위기가 지역 스포츠 활동에 끼친 장기적인 영향을 평가합니다.
- **아이디어의 우수한 점**: 팬데믹 상황에서의 스포츠 활동 변화를 이해함으로써, 비슷한 상황에서의 대응 전략 개발에 도움을 줄 수 있습니다.
- **아이디어 구현의 문제점**: 시계열 데이터의 접근성 및 연속성이 필수적이며, 이를 정확히 분석하기 위해선 추가적인 배경 정보와 데이터가 필요할 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15023796
- 목록명: 부산광역시교육청 부산광역시해운대교육지원청_공동체육관 정보

### 3. 지역 체육 프로그램의 접근성 및 다양성 평가
- **아이디어 설명**: 해당 데이터를 기반으로 부산 해운대 지역의 공동체육관 프로그램의 접근성과 다양성을 평가합니다. 프로그램이 다양한 연령층과 능력을 가진 인구에게 얼마나 잘 서비스되고 있는지를 분석하고, 개선할 수 있는 영역을 식별합니다.
- **아이디어의 우수한 점**: 모든 시민이 체육 활동에 참여할 수 있는 기회를 제공함으로써 지역사회의 포괄성과 건강을 증진할 수 있습니다. 또한, 지역사회의 체육 시설 및 프로그램을 개선하는 데 필요한 구체적인 권장 사항을 제공할 수 있습니다.
- **아이디어 구현의 문제점**: 프로그램의 접근성과 다양성을 평가하기 위해서는 상세한 참여자 데이터와 그 배경 정보가 필요합니다. 데이터가 불충분하거나 비공개일 수 있으며, 민감한 정보를 취급함에 있어 개인정보 보호 문제도 고려해야 합니다.

**데이터 목록키와 목록명**
- 목록키: 15023796
- 목록명: 부산광역시교육청 부산광역시해운대교육지원청_공동체육관 정보

이 데이터는 부산 해운대 지역의 공동체육관 프로그램의 참여 정보, 다양성, 접근성 평가 등을 분석하는 데 활용될 수 있습니다.

## 사회복지-고용노동

In [16]:
filename = '사회복지-고용노동'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 학생 진로 지원 서비스의 효과성 분석
- **아이디어 설명**: "광주광역시교육청_진로진학정보센터" 데이터를 활용하여, 제공되는 진로 및 진학 지원 서비스가 학생들의 진로 결정 및 대입 성공에 어떤 영향을 미치는지 분석합니다. 서비스 접근성, 이용 빈도, 학생 및 학부모의 만족도 등을 조사합니다.
- **아이디어의 우수한 점**: 교육정책의 효과성을 실제 데이터로 평가하여, 필요한 개선점을 발견하고 정책 조정에 기여할 수 있습니다.
- **아이디어 구현의 문제점**: 진로 결정에 영향을 미치는 요인이 다양하므로, 특정 서비스의 영향력을 정확히 평가하기 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 3036292
- 목록명: 광주광역시교육청_진로진학정보센터

### 2. 전문연구 정보 활용과 채용 시장 분석
- **아이디어 설명**: "한국연구재단_전문연구정보(NRIC) 조회 오픈 API 서비스" 데이터를 분석하여, 전문 연구 정보가 채용 시장에 어떤 영향을 미치는지 조사합니다. 특히 연구 분야의 전문가 수요와 공급, 채용 트렌드 변화를 분석합니다.
- **아이디어의 우수한 점**: 고급 인재의 수요와 공급을 이해하고, 미래의 채용 시장을 예측하는 데 도움을 줄 수 있습니다.
- **아이디어 구현의 문제점**: 채용 시장의 다양한 요인과 연구 정보 사이의 직접적인 상관관계를 명확히 파악하기 어려울 수 있습니다.

**데이터 목록키와 목록명**
- 목록키: 15088749
- 목록명: 한국연구재단_전문연구정보(NRIC) 조회 오픈 API 서비스

### 3. 교육 자료의 디지털화와 접근성 향상
- **아이디어 설명**: 위의 데이터셋을 활용하여 교육청에서 제공하는 다양한 디지털 교육 자료의 접근성과 활용도를 평가합니다. 특히 원격 교육 및 온라인 자료의 이용 증가 추세를 분석하고, 이에 따른 교육적 결과를 조사합니다.
- **아이디어의 우수한 점**: 디지털 교육 자료의 효과적인 사용은 교육의 질을 높이고, 교육 접근성을 개선하는 데 중요합니다. 이 분석을 통해 교육 자료의 배포와 활용 방안을 최적화할 수 있습니다.
- **아이디어 구현의 문제점**: 온라인 교육 자료의 이용 효과를 평가하기 위해서는 학습 결과와의 직접적인 연관성을 증명하기 위한 장기적인 데이터 추적과 복잡한 데이터 분석이 필요합니다.

**데이터 목록키와 목록명**
- 데이터 목록키와 목록명은 이 아이디어에서는 특정 데이터셋을 지정하지 않았습니다. 이 아이디어는 제공된 두 데이터셋을 통합하여 교육 자료의 디지털화와 접근성에 초점을 맞추며, 관련 데이터를 분석하는 데 적용될 수 있습니다.

이 세 가지 아이디어는 "사회복지-고용노동" 데이터를 활용하여 교육 및 연구 분야에서의 정보 접근성, 진로 지원 서비스의 효과성, 그리고 전문 연구 정보의 채용 시장에 미치는 영향을 다양한 관점에서 분석합니다. 이를 통해 관련 정책과 프로그램을 보다 효과적으로 개선하고, 교육 및 연구 분야에서의 인사이트를 제공할 수 있습니다.

## 사회복지-공적연금

In [17]:
filename = '사회복지-공적연금'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 지역별 퇴직자 발생 추이 분석
- **아이디어 설명**: 사립학교 교직원의 지역별 퇴직자 발생 추이를 분석하여, 어느 지역에서 퇴직자가 가장 많이 발생하는지 그리고 그 이유를 추측해볼 수 있습니다.
- **우수한 점**: 지역별 교육 인력의 흐름을 이해할 수 있으며, 교육 정책이나 학교 운영에 대한 시사점을 제공할 수 있습니다.
- **구현의 문제점**: 퇴직자 발생의 원인을 구체적으로 알아보기 위해서는 추가적인 정보가 필요할 수 있습니다.

### 2. VOC 민원 데이터 분석
- **아이디어 설명**: 사립학교 교직원 연금공단에 접수된 VOC(고객의 소리) 민원 중 상위 5개 민원의 월별 발생 건수를 분석합니다. 어떤 문제가 가장 많이 제기되는지 확인하고 해결 방안을 모색할 수 있습니다.
- **우수한 점**: 민원 데이터를 통해 연금 공단의 주요 문제점을 파악하고 개선 방향을 제시할 수 있습니다.
- **구현의 문제점**: 민원의 원인과 문제 해결 방안을 찾기 위해 더 깊은 분석과 이해가 필요합니다.

### 3. 급여 미청구자 관리 현황 분석
- **아이디어 설명**: 급여를 청구하지 않은 사립학교 교직원의 현황을 분석하여, 청구하지 않는 이유와 이들에 대한 관리 방안을 연구합니다.
- **우수한 점**: 급여 미청구 문제를 해결함으로써 교직원들의 권리 보호와 만족도를 높일 수 있습니다.
- **구현의 문제점**: 개인의 상황에 따라 다양한 이유가 있을 수 있어, 정확한 원인 분석이 어려울 수 있습니다.

### 필요한 데이터 목록키 및 목록명
1. **목록키**: 15125546, **목록명**: 사립학교교직원연금공단_지역별 퇴직자 발생 추이
2. **목록키**: 15124513, **목록명**: 사립학교교직원연금공단_VOC 상위 5개 민원별 월별 발생 건수
3. **목록키**: 15081264, **목록명**: 사립학교교직원연금공단_급여 미청구자 관리 현황

## 사회복지-보육·가족및여성

In [18]:
filename = '사회복지-보육·가족및여성'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list


### 1. 육아 휴직 이용률 분석
- **아이디어 설명**: 한국장학재단에서 육아 휴직을 이용하는 남녀 근로자의 비율을 분석합니다. 성별에 따른 이용률 차이를 시각적으로 표현해 보고, 그 원인을 탐색해볼 수 있습니다.
- **우수한 점**: 성별에 따른 직장 내 문화나 정책의 차이를 이해할 수 있고, 육아 휴직에 대한 인식 개선에 기여할 수 있습니다.
- **구현의 문제점**: 성별 이외의 다른 변수(연령, 직급 등)의 영향을 분석하기 위한 데이터가 부족할 수 있습니다.

### 2. 육아 휴직 기간과 근무 만족도 연관성 분석
- **아이디어 설명**: 육아 휴직을 사용한 근로자의 휴직 기간과 복귀 후 근무 만족도 사이의 관계를 분석합니다. 이 데이터를 활용하여 근로자가 어떻게 휴직 기간을 설정하는 것이 복귀 후 만족도에 긍정적인지 조사할 수 있습니다.
- **우수한 점**: 휴직 기간을 어떻게 설정하면 복귀 후 근로자의 만족도가 높아질 수 있는지에 대한 인사이트를 제공할 수 있습니다.
- **구현의 문제점**: 만족도를 측정하기 위한 정확한 데이터가 필요하며, 이를 얻기 위해서는 추가적인 설문조사나 데이터 수집이 필요할 수 있습니다.

### 3. 육아 휴직 이용 추세 분석
- **아이디어 설명**: 최근 몇 년간 한국장학재단의 육아 휴직 이용 추세를 분석합니다. 시간에 따라 변화하는 휴직 이용률을 그래프로 그려보고, 그 원인을 탐구해볼 수 있습니다.
- **우수한 점**: 시간에 따른 변화를 통해 정책이나 사회적 인식의 변화를 이해할 수 있습니다.
- **구현의 문제점**: 여러 해에 걸친 데이터가 필요하며, 단기간 내에 변화를 분석하기에는 데이터가 부족할 수 있습니다.

### 필요한 데이터 목록키 및 목록명
- **목록키**: 15080684
- **목록명**: 한국장학재단_육아휴직_현황

## 사회복지-사회복지일반

In [19]:
filename = '사회복지-사회복지일반'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 사회정책 대표누리집 추진계획 분석
- **아이디어 설명**: 교육부의 사회정책 대표누리집에서 제공하는 추진계획을 분석하여, 교육 관련 사회정책의 주요 방향과 초점을 파악합니다. 정책별 추진 배경과 목표를 이해하고, 그 효과를 예측해볼 수 있습니다.
- **우수한 점**: 사회정책의 이해를 돕고, 정책이 학생과 교육기관에 미치는 영향을 탐색할 수 있습니다.
- **구현의 문제점**: 정책의 세부 진행 상황이나 구체적인 영향을 측정하기 위한 데이터가 부족할 수 있습니다.

  **필요한 데이터 목록키 및 목록명**
  - **목록키**: 15118808
  - **목록명**: 교육부_사회정책 대표누리집_추진계획

### 2. 회차별 안건내용 분석
- **아이디어 설명**: 교육부가 개최하는 회의에서 다루어진 주요 안건들의 내용을 분석하여, 정책 결정 과정에서 중점을 두는 사항들을 파악합니다. 이를 통해 정책의 변화와 발전 방향을 예측할 수 있습니다.
- **우수한 점**: 정책의 주요 이슈와 추진 방향을 이해함으로써, 교육 정책의 흐름을 파악하는데 도움을 줄 수 있습니다.
- **구현의 문제점**: 안건의 내용만으로는 구체적인 실행 방법이나 세부 결과를 알기 어렵습니다.

  **필요한 데이터 목록키 및 목록명**
  - **목록키**: 15118809
  - **목록명**: 교육부_사회정책 대표누리집_회차별 안건내용

### 3. 사회공헌 기부현황 분석
- **아이디어 설명**: 한국장학재단의 사회공헌 기부현황을 분석하여, 어떤 기부가 주로 이루어지고 있는지, 기부의 추세가 어떻게 변화하는지 파악합니다. 또한, 기부가 교육과 학생들에게 어떤 영향을 미치는지 연구할 수 있습니다.
- **우수한 점**: 공공기관의 사회공헌 활동을 이해하고, 이를 통한 사회적 가치 창출의 중요성을 학습할 수 있습니다.
- **구현의 문제점**: 기부의 직접적인 영향을 분석하기 위해서는 추가적인 데이터나 연구가 필요할 수 있습니다.

  **필요한 데이터 목록키 및 목록명**
  - **목록키**: 15084567
  - **목록명**: 한국장학재단_사회공헌 기부현황

## 사회복지-주택

In [20]:
filename = '사회복지-주택'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 창업기숙사 입주율 분석 및 개선 아이디어
- **설명**: 창업기숙사의 입주율을 조사하고, 입주율이 낮은 이유와 이를 개선할 수 있는 방안을 제안한다. 예를 들어, 학생들이 창업기숙사에 입주하지 않는 이유를 설문조사를 통해 파악하고, 그 결과를 바탕으로 개선책을 모색한다.
- **장점**: 창업기숙사의 활용도를 높이고, 더 많은 학생이 창업을 지원받을 수 있도록 돕는다.
- **문제점**: 정확한 설문조사와 데이터 분석이 필요하며, 설문 결과에 따른 실행 가능한 솔루션을 제시하기가 어려울 수 있다.
- **목록키**: 15115694
- **목록명**: 한국장학재단_창업기숙사 현황

### 2. 창업기숙사의 지원 프로그램 효과성 분석
- **설명**: 창업기숙사에서 제공하는 다양한 창업 지원 프로그램의 효과성을 분석한다. 프로그램 참가자들의 만족도 조사와 창업 성공률을 측정하여, 가장 효과적인 프로그램을 파악한다.
- **장점**: 효과적인 창업 지원 프로그램을 확인하고, 이를 통해 더 나은 창업 지원 환경을 조성할 수 있다.
- **문제점**: 다양한 프로그램과 창업자의 개별적인 차이를 고려한 복잡한 데이터 분석이 요구된다.
- **목록키**: 15115694
- **목록명**: 한국장학재단_창업기숙사 현황

### 3. 창업기숙사 입주생의 지역적 배경 분석
- **설명**: 창업기숙사에 입주하는 학생들의 지역적 배경을 분석하여, 특정 지역에서 창업 지원을 받기 위해 오는 학생들이 많은지 조사한다. 또한 지역별로 창업 기숙사의 수요를 파악한다.
- **장점**: 지역별 창업 지원 수요를 이해하고, 지역 균형 발전을 위한 정책 제안이 가능하다.
- **문제점**: 입주자의 개인 정보 보호와 데이터 접근성에 대한 문제가 발생할 수 있다.
- **목록키**: 15115694
- **목록명**: 한국장학재단_창업기숙사 현황

## 사회복지-취약계층지원

In [21]:
filename = '사회복지-취약계층지원'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 기부자 혜택과 기부금 증가율의 상관관계 분석
- **설명**: 한국장학재단에서 제공하는 기부자 혜택이 기부금 증가에 얼마나 영향을 미치는지 분석한다. 기부자 혜택이 좋을수록 기부금이 더 많이 증가하는지 데이터를 통해 확인한다.
- **장점**: 기부자 혜택의 효과를 실질적으로 평가할 수 있으며, 더 효과적인 기부자 유치 전략을 개발할 수 있다.
- **문제점**: 다른 요인들도 기부금에 영향을 미칠 수 있어, 혜택과 기부금 간의 직접적인 인과관계를 확립하기 어려울 수 있다.
- **목록키**: 15099444
- **목록명**: 한국장학재단_기부자 혜택

### 2. 기부자 예우 프로그램의 인지도 조사 및 개선 제안
- **설명**: 기부자들이 현재 받고 있는 예우 및 혜택에 대한 인지도를 조사하고, 기부자들의 만족도를 높일 수 있는 방안을 제안한다.
- **장점**: 기부자 만족도를 높여 장기적인 기부 유지를 도모할 수 있으며, 기부자 예우 프로그램의 효율성을 개선할 수 있다.
- **문제점**: 만족도 조사와 피드백 수집을 위한 체계적인 접근 방법과 자원이 필요하다.
- **목록키**: 15099444
- **목록명**: 한국장학재단_기부자 혜택

### 3. 다양한 기부자 혜택 모델 비교 분석
- **설명**: 다른 장학재단이나 기부 기관에서 제공하는 기부자 혜택과 한국장학재단의 혜택을 비교 분석하여, 어떤 혜택이 가장 효과적인지 조사한다.
- **장점**: 보다 효과적인 기부자 유치 전략을 개발할 수 있고, 기부자들에게 더 매력적인 혜택을 제공할 수 있다.
- **문제점**: 다른 기관의 자료 수집에 있어 접근성 및 정보의 정확성 문제가 발생할 수 있다.
- **목록키**: 15099444
- **목록명**: 한국장학재단_기부자 혜택

## 일반공공행정-법제

In [22]:
filename = '일반공공행정-법제'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 한국장학재단 내규의 변화 추적 및 분석
- **설명**: 한국장학재단의 내규가 시간에 따라 어떻게 변화해 왔는지 연도별로 추적하고 분석한다. 내규 변경이 장학재단 운영에 어떤 영향을 미쳤는지 사례를 통해 살펴본다.
- **장점**: 기관의 정책 변화가 실제 운영에 미치는 영향을 이해하고, 정책 결정 과정의 중요성을 배울 수 있다.
- **문제점**: 내규의 구체적인 변경 내용과 그 영향을 정확히 파악하기 위해선 상세한 문서 분석과 이해가 필요하다.
- **목록키**: 15083931
- **목록명**: 한국장학재단_내규현황

### 2. 내규 준수와 학생 만족도의 관계 분석
- **설명**: 한국장학재단의 내규가 학생들의 장학금 수령 경험에 어떤 영향을 미치는지 분석한다. 내규 준수가 학생 만족도에 미치는 영향을 설문조사를 통해 조사한다.
- **장점**: 내규가 실제 이해당사자인 학생들에게 어떤 영향을 미치는지 이해하고, 학생 중심의 정책 개선을 제안할 수 있다.
- **문제점**: 학생들의 응답을 수집하고 분석하는 데 필요한 자원과 시간이 많이 소요될 수 있다.
- **목록키**: 15083931
- **목록명**: 한국장학재단_내규현황

### 3. 한국장학재단 내규의 교육 정책에 대한 영향 분석
- **설명**: 한국장학재단의 내규가 교육 정책, 특히 장학금 제도에 어떤 영향을 미치는지 분석한다. 내규의 주요 변경 사항과 그 배경을 조사하고, 교육 정책에 미친 영향을 평가한다.
- **장점**: 정책 결정자들이 어떤 고려 사항을 바탕으로 정책을 개선하려고 했는지 이해할 수 있다.
- **문제점**: 복잡한 정책 문서를 해석하고 그 영향력을 분석하기 위해서는 상당한 정책 이해도가 요구된다.
- **목록키**: 15083931
- **목록명**: 한국장학재단_내규현황

## 일반공공행정-일반행정

In [23]:
filename = '일반공공행정-일반행정'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 교원 소청 심사 결정 추이 분석
- **설명**: 연도별로 교원 소청 심사의 접수 건수와 결정 건수(인용, 기각)의 추이를 분석하여, 교육 정책이나 교원 근무 환경의 변화를 추론해 본다.
- **장점**: 교육 정책의 변화가 교원들의 고충 해결에 어떤 영향을 미치는지 이해할 수 있으며, 교육 정책에 대한 피드백을 제공할 수 있다.
- **문제점**: 복잡한 데이터를 이해하고 분석하기 위해 통계적인 기법과 데이터 해석 능력이 필요하다.
- **목록키**: 15121095
- **목록명**: 교육부 교원소청심사위원회_교원소청심사현황(연도별)

### 2. 폐교 재산 활용도 및 매각 현황 분석
- **설명**: 폐교 재산의 보유 현황 및 매각 상황을 조사하여, 폐교 재산이 어떻게 활용되고 있는지 분석한다. 매각된 폐교의 활용 사례를 통해 효율적인 폐교 재산 활용 방안을 제안한다.
- **장점**: 지역 사회에서 폐교 재산을 활용하여 경제적, 사회적 이익을 창출하는 방법을 탐색할 수 있다.
- **문제점**: 다양한 지역과 매각 상황에 대한 데이터를 일관되게 분석하는 것이 도전적일 수 있다.
- **목록키**: 15053775, 15053776
- **목록명**: 경상북도교육청 경상북도봉화교육지원청_폐교보유현황, 경상북도교육청 경상북도봉화교육지원청_폐교매각현황

### 3. 도서관 신착 도서 목록 분석 및 추천 시스템 개발
- **설명**: 대구광역시립 동부 도서관의 신착 도서 목록을 분석하여, 어떤 종류의 도서가 주로 도입되고 있는지 파악한다. 이를 바탕으로 독자의 취향에 맞는 도서 추천 시스템을 제안한다.
- **장점**: 도서관 이용자들이 자신의 관심사에 맞는 책을 쉽게 찾을 수 있도록 돕고, 도서관 이용률을 증가시킬 수 있다.
- **문제점**: 도서의 종류와 도서의 분류 기준을 정확히 이해하고 적용하기 위한 상세한 데이터 분석이 필요하다.
- **목록키**: 15073852
- **목록명**: 대구광역시교육청 대구광역시립동부도서관_신착도서 목록

## 일반공공행정-지방행정·재정지원

In [24]:
filename = '일반공공행정-지방행정·재정지원'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 지방교육재정의 효율성 분석
- **설명**: 각 시도교육청의 예산과 결산 정보를 분석하여, 교육 예산이 어떻게 사용되고 있는지 파악한다. 특히, 예산 사용의 효율성을 중점적으로 분석하여, 예산이 교육의 질 향상에 어떻게 기여하고 있는지 조사한다.
- **장점**: 예산 사용의 투명성과 효율성을 이해하고, 교육재정의 개선 방안을 제시할 수 있다.
- **문제점**: 복잡한 재정 데이터를 해석하고 분석하는 데 상당한 양의 정보와 이해가 필요하다.
- **목록키**: 15052877
- **목록명**: 교육부_지방교육재정정보 데이터셋

### 2. 일일 교육 재정 수입지출 현황 분석
- **설명**: 일일 수입지출 데이터를 분석하여, 교육 예산이 일상적으로 어떻게 집행되고 있는지 파악한다. 이를 통해 예산 집행의 시기적 적절성 및 긴급 필요에 대응하는 능력을 평가한다.
- **장점**: 교육청의 예산 집행 패턴을 이해하고, 예산 관리의 적시성 및 효율성을 평가할 수 있다.
- **문제점**: 일일 데이터의 대량 분석이 필요하며, 데이터의 지속적인 업데이트가 요구된다.
- **목록키**: 15052879
- **목록명**: 교육부_지방교육재정정보 일일수입지출현황 당해년도

### 3. 대안교육운영 재정지원 분석
- **설명**: 대안교육 프로그램에 대한 재정지원 내역을 분석하여, 재정 지원이 대안교육의 운영과 발전에 어떤 영향을 미치고 있는지 조사한다. 지원의 충분성과 효과성을 중점적으로 평가한다.
- **장점**: 대안교육에 대한 재정 지원의 중요성을 이해하고, 지원 정책의 개선점을 찾을 수 있다.
- **문제점**: 재정지원의 직접적인 교육 효과를 측정하기 위해서는 추가적인 교육 성과 데이터가 필요하다.
- **목록키**: 15052878
- **목록명**: 교육부_대안교육운영 재정지원

## 통신-방송통신

In [25]:
filename = '통신-방송통신'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. ICT 연구동향 분석
- **설명**: 한국연구재단에서 발행한 ICT 관련 연구결과보고서를 분석하여 최근 몇 년간의 연구 동향을 파악한다. 어떤 ICT 분야가 가장 활발히 연구되고 있는지, 주요 연구 주제는 무엇인지 조사한다.
- **장점**: ICT 분야의 최신 연구 경향을 이해하고, 향후 연구 방향이나 관심 분야를 예측할 수 있다.
- **문제점**: 연구 보고서를 이해하기 위해 기술적인 용어와 개념을 정확히 알아야 할 필요가 있다.
- **목록키**: 15036875
- **목록명**: 한국연구재단_연구결과보고서 정보

### 2. ICT 통계 자료를 이용한 기술 발전 추이 분석
- **설명**: 한국연구재단에서 제공하는 ICT 통계 간행물 정보를 분석하여, 시간에 따른 기술 발전 추이와 산업의 변화를 조사한다.
- **장점**: 과학기술의 진보가 어떻게 사회와 산업에 영향을 미치는지 시각적으로 이해할 수 있으며, 데이터 분석 능력을 키울 수 있다.
- **문제점**: 복잡한 통계 자료를 해석하기 위해선 기본적인 통계 지식이 필요하다.
- **목록키**: 15036876
- **목록명**: 한국연구재단_ ICT통계간행물 정보

### 3. ICT 분야 최신 동향 정보의 중요성 교육
- **설명**: 한국연구재단에서 제공하는 최신 ICT 동향 정보를 학교 커리큘럼에 통합하여, 학생들이 현재 ICT 산업에서 일어나는 중요한 변화와 동향을 실시간으로 학습할 수 있는 프로그램을 제안한다.
- **장점**: 학생들이 실시간 시장과 기술 변화에 대응하는 능력을 키우며, 미래 지향적인 사고방식을 개발할 수 있다.
- **문제점**: 최신 정보를 항상 업데이트하고, 교육 프로그램에 통합하는 데 추가적인 자원과 노력이 필요하다.
- **목록키**: 15036878
- **목록명**: 한국연구재단_ICT동향정보

## 통일·외교-외교

In [26]:
filename = '통일·외교-외교'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 유네스코 국제규범의 영향력 분석
- **설명**: 유네스코에서 발표한 국제규범이 각국의 정책에 어떤 영향을 미치는지 사례를 조사하여 분석한다. 특히, 교육, 과학, 문화 분야에서의 규범 적용 사례를 중심으로 조사한다.
- **장점**: 국제 기구의 정책이 실제로 세계 각국에 어떤 영향을 미치는지 이해하고, 글로벌 이슈에 대한 관심을 높일 수 있다.
- **문제점**: 다양한 국가와 정책의 특성을 이해하고 분석하는 데 상당한 시간과 노력이 필요하다.
- **목록키**: 3059534
- **목록명**: 유네스코한국위원회_유네스코국제규범 및 유네스코한국위원회_발간 자료

### 2. 유네스코 문화 유산의 보존 중요성 교육
- **설명**: 유네스코에서 지정하는 세계 문화 유산의 중요성을 알리고, 보존 활동에 대해 학습하는 교육 프로그램을 제안한다. 이를 통해 학생들이 문화 유산의 가치와 보존 필요성을 이해할 수 있도록 한다.
- **장점**: 문화적 가치와 세계 문화에 대한 인식을 넓히며, 보존 활동의 중요성을 인식하게 한다.
- **문제점**: 구체적인 보존 활동과 관련된 데이터나 정보의 접근성이 제한적일 수 있다.
- **목록키**: 3059534
- **목록명**: 유네스코한국위원회_유네스코국제규범 및 유네스코한국위원회_발간 자료

### 3. 유네스코 교육 프로그램의 효과성 분석
- **설명**: 유네스코에서 지원하는 교육 프로그램이 전 세계 어린이와 청소년들에게 미치는 영향을 분석한다. 특히, 저개발국가에서의 교육 접근성 향상과 교육의 질 개선에 초점을 맞춘다.
- **장점**: 국제적인 교육 지원이 실질적인 교육 격차 해소에 어떻게 기여하는지 이해할 수 있다.
- **문제점**: 국제적인 데이터 수집과 분석에 있어 어려움이 따르며, 통계적인 처리가 필요하다.
- **목록키**: 3059534
- **목록명**: 유네스코한국위원회_유네스코국제규범 및 유네스코한국위원회_발간 자료

## 해양수산-해양수산·어촌

In [27]:
filename = '해양수산-해양수산·어촌'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 지역별 어선 검사 현황과 안전성 비교 분석
- **설명**: 각 지역별로 어선의 검사 현황을 조사하고, 그 결과를 바탕으로 지역별 어선의 안전성을 비교 분석한다. 어선의 검사 통과율과 사고 발생률을 연관지어 본다.
- **장점**: 안전한 어선 운영의 중요성을 이해하고, 특정 지역에서 안전 관리가 잘되고 있는지 평가할 수 있다.
- **문제점**: 사고 발생 데이터와 연계하여 분석하기 위해서는 추가 데이터 수집이 필요할 수 있다.
- **목록키**: 3045157
- **목록명**: 선박안전기술공단_선박 검사대상 현황

### 2. 선박 검사 기준 교육 및 인식 개선 캠페인 아이디어
- **설명**: 선박 검사의 중요성을 알리기 위해 학교에서 선박 안전 기준에 대한 교육 캠페인을 실시한다. 학생들이 선박 안전의 중요성을 배우고, 지역 사회에도 그 정보를 전파한다.
- **장점**: 학생들이 선박 안전에 대한 인식을 높이고, 지역 사회의 안전 문화를 강화할 수 있다.
- **문제점**: 실제 선박을 이용하지 않는 학생들에게는 관련성이 떨어질 수 있다.
- **목록키**: 3045157
- **목록명**: 선박안전기술공단_선박 검사대상 현황

### 3. 어선 검사 데이터를 활용한 안전 개선 제안
- **설명**: 현재 어선의 검사 데이터를 분석하여 어떤 요소에서 불합격이 많이 발생하는지 파악하고, 이를 통해 어선의 안전성을 향상시킬 수 있는 구체적인 개선 방안을 제안한다.
- **장점**: 실질적인 데이터를 바탕으로 한 제안으로 어선 운영의 안전을 직접적으로 향상시킬 수 있다.
- **문제점**: 전문적인 해양 지식이 요구되며, 복잡한 기술적 문제에 대한 이해가 필요하다.
- **목록키**: 3045157
- **목록명**: 선박안전기술공단_선박 검사대상 현황

## 환경-대기

In [29]:
filename = '환경-대기'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 학교별 온실가스 감축 경진대회 아이디어
- **설명**: 학교별로 온실가스 배출량을 조사하여 어떤 학교가 가장 많은 온실가스를 감축했는지 경쟁을 만들어보자. 감축량은 전년도 대비 감축 비율로 계산한다.
- **장점**: 학생들이 환경 보호에 관심을 갖고 자발적으로 에너지 절약을 실천할 수 있는 계기를 마련할 수 있다.
- **문제점**: 각 학교의 초기 데이터가 필요하며, 모든 학교에서 동일한 조건으로 데이터를 수집해야 공정한 경쟁이 가능하다.

### 2. 교육시설의 친환경 전환을 위한 분석 아이디어
- **설명**: 현재 학교에서 사용하는 전기나 난방 시스템이 환경에 얼마나 영향을 미치는지 분석하고, 태양광 등 친환경 에너지로 전환할 경우의 이점을 연구한다.
- **장점**: 장기적으로 비용 절감과 환경 보호 두 가지 측면에서 이익을 볼 수 있다.
- **문제점**: 초기 투자 비용이 많이 들고, 구체적인 기술적 분석이 필요하다.

### 3. 온실가스 배출과 감축의 연간 추세 분석
- **설명**: 매년 사립학교교직원연금공단의 온실가스 배출량과 감축 실적을 추적하여 추세를 분석해보자. 이를 통해 효과적인 감축 정책을 제안할 수 있다.
- **장점**: 데이터를 기반으로 한 실질적인 정책 제안이 가능하며, 정책의 성공적인 시행을 위한 근거 자료를 제공할 수 있다.
- **문제점**: 데이터의 연속성과 정확성이 중요하며, 여러 해에 걸친 데이터가 필요하다.

### 관련 데이터의 목록키와 목록명
1. **목록키**: 15060554
   - **목록명**: 사립학교교직원연금공단_온실가스 감축실적
2. **목록키**: 15062414
   - **목록명**: 사립학교교직원연금공단_온실가스 배출량

## 환경-폐기물

In [28]:
filename = '환경-폐기물'
item_list = list(pd.read_csv(f'{filename}.csv')['목록명'])
# item_list

### 1. 학교별 폐기물 처리 비용 절감 대회 아이디어
- **설명**: 학교에서 발생하는 폐기물의 양을 측정하고, 어떤 방법으로 가장 효율적으로 폐기물을 처리할 수 있는지 경쟁을 만들어보자. 각 학교는 폐기물을 줄이기 위한 창의적인 아이디어를 제안하고 실천할 수 있다.
- **장점**: 학생들이 폐기물 관리와 비용 절감의 중요성을 배우며 환경 보호에 기여할 수 있다.
- **문제점**: 폐기물의 정확한 양을 측정하고 감소시킨 결과를 정량화하는 데 어려움이 있을 수 있다.
- **목록키**: 15106855
- **목록명**: 부경대학교_폐기물 처리내역

### 2. 폐기물 분리수거 효과 분석 아이디어
- **설명**: 학교에서 실시하는 분리수거가 얼마나 효과적인지를 분석해보자. 폐기물의 종류별로 얼마나 잘 분리되고 있는지, 재활용률은 얼마나 되는지 조사한다.
- **장점**: 분리수거의 중요성을 이해하고, 학생들이 적극적으로 재활용을 실천하는 습관을 기를 수 있다.
- **문제점**: 실제 분리수거 효과를 측정하기 위해선 추가적인 데이터 수집과 현장 조사가 필요할 수 있다.
- **목록키**: 15106855
- **목록명**: 부경대학교_폐기물 처리내역

### 3. 친환경 소재 사용으로의 전환 아이디어
- **설명**: 학교에서 사용하는 일회용품의 재질을 친환경 소재로 변경했을 때, 폐기물 처리 비용 및 환경적 영향을 분석해보자. 플라스틱 대신 생분해성 재료 사용의 효과를 비교한다.
- **장점**: 환경 보호에 실질적으로 기여할 수 있으며, 친환경 소재의 필요성을 알릴 수 있다.
- **문제점**: 친환경 소재의 비용이 일반 소재보다 높을 수 있어, 비용 효과 분석이 필요하다.
- **목록키**: 15106855
- **목록명**: 부경대학교_폐기물 처리내역